src/univariate/discrete/binomial.jl

```diff
@@ -164,8 +179,8 @@ function suffstats{T<:Integer}(::Type{Binomial}, n::Integer, x::AbstractArray{T}
 end
 
 function suffstats{T<:Integer}(::Type{Binomial}, n::Integer, x::AbstractArray{T}, w::AbstractArray{Float64})
-    ns = 0.
-    ne = 0.
+    ns = 0
+    ne = 0
```

Comment:
I put them back. I plan to go through and fix the`suffstats` routines in a later PR. For now, we've left the fitting alone. Again, this is overzealous `.` removal.